src/pages/home/report/ReportActionsView.js

```diff
@@ -200,6 +222,13 @@ class ReportActionsView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        if (this.props.reportActionID && this.props.reportActionID !== prevProps.reportActionID && this.props.reportID === prevProps.reportID) {
```

Comment:
Using a change in the `reportActionID` seems fine when it actually changes. But this won't work if there is no change and the user navigated to the same action.

Reply:
Ah hah, yes i just mentioned above about the same in the previous comment. Its part of known limitation so i was hoping to discuss plan of action when we're working on this.
Mostly because i haven't decided if the best way is to update the code here or pop out the reportActionID in the URL.